CHANGELOG.md

```diff
@@ -41,10 +41,13 @@ Consider migrating to `stream-chat-android-ui-components` or `stream-chat-androi
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added new extension function `ChatClient::deleteChannel`. [#3007](https://github.com/GetStream/stream-chat-android/pull/3007)
 
 ### ⚠️ Changed
+- Deprecated `ChatDomain::deleteChannel` in favour of `ChatClient::deleteChannel`. [#3007](https://github.com/GetStream/stream-chat-android/pull/3007)
 
 ### ❌ Removed
+- `DeleteChannel` usecase is now removed. [#3007](https://github.com/GetStream/stream-chat-android/pull/3007)
```

Comment:
`DeleteChannel` use case was internal so we don't need to mention that here

Reply:
Done